frontend/src/routes/Governance/components/PolicyActionDropdown.tsx

```diff
@@ -184,15 +192,21 @@ export function PolicyActionDropdown(props: {
           })
         },
         rbac:
-          policyRemediationAction === 'inform'
+          policyRemediationAction === REMEDIATION_ACTION.INFORM
             ? undefined
             : [rbacPatch(PolicyDefinition, item.policy.metadata.namespace)],
       },
       {
         id: 'enforce-policy',
         text: t('policy.table.actions.enforce'),
-        tooltip: policyRemediationAction === 'enforce' ? t('Already enforcing') : t('Enforce policy'),
-        isAriaDisabled: policyRemediationAction === 'enforce',
```

Comment:
Similar comment as before. The existing code was fine. The override shouldn't influence your ability to set the remediationAction on the policy.